server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -2,12 +2,15 @@
 [[crossdc-mode]]
 === Cross-Datacenter Replication Mode
 
-NOTE: Cross-Datacenter Replication Mode is Technology Preview and is not fully supported.
-
 :tech_feature_name: Cross-Datacenter Replication Mode
 :tech_feature_disabled: false
 include::../templates/techpreview.adoc[]
 
+[NOTE]
+====
```

Comment:
This shouldn't be needed as it's already in techpreview.adoc. Looks like for some reason that is not working.